psqlgraph/query.py

```diff
@@ -553,7 +553,20 @@ def has_sysan(self, keys):
 
 
 def is_list_prop(entity, prop):
-    if prop not in entity.__pg_properties__:
-        raise KeyError("{} has not attribute {}".format(entity.__class__, prop))
-    return list in entity.__pg_properties__[prop]
-
+    """Determine if a property on an entity is a list type.
+
+    The Node type is a special case because it has no properties but its
+    subclasses do have them. It's also special in that it's the only node
+    type with subclasses. Going through the subclasses to find a matching
```

Comment:
This is definitely good for now 😄 . Something to think about: I did read this comment and I thought "what if we end up adding some other node type with subclasses? Are there any assumptions that will break in this code?" Then I read your code and I think it's futureproof (although, is that true, would we need to check the subclasses on such a node type?). If it is futureproof, I think you can remove this comment.

Reply:
Prior to pushing this PR I checked my dev-repl to see if the Node's subclasses had subclasses themselves, but they all turned up empty. I think the way we have psqlgraph/gdcdatamodel setup It's unlikely that we would end up in a situation where that would be true in the future.